youtube_dl/extractor/streamcz.py

```diff
@@ -93,4 +93,15 @@ def _real_extract(self, url):
             'description': data.get('web_site_text'),
             'duration': int_or_none(data.get('duration')),
             'view_count': int_or_none(data.get('views')),
+            'subtitles': self.extract_subtitles(data.get('subtitles_srt')),
         }
+
+    @staticmethod
+    def _get_subtitles(subs):
```

Comment:
This methods is only used when subtitles' extraction depends on additional expensive work, i.e. network I/O. In this case subtitles are already extracted along with formats and using this method does not make any sense.